src/StackExchange.Redis/ChannelMessageQueue.cs

```diff
@@ -48,6 +48,16 @@ internal ChannelMessage(ChannelMessageQueue queue, in RedisChannel channel, in R
         /// The value that was broadcast
         /// </summary>
         public RedisValue Message { get; }
+
+        /// <summary>
+        /// Checks if 2 messages are .Equal()
+        /// </summary>
+        public static bool operator ==(ChannelMessage left, ChannelMessage right) => left.Equals(right);
+
+        /// <summary>
+        /// Checks if 2 messages are not .Equal()
+        /// </summary>
+        public static bool operator !=(ChannelMessage left, ChannelMessage right) => !(left == right);
```

Comment:
feels like an unnecessary indirection here; OK, it'll probable be inlined etc, but why not `!left.Equals(right)` ?